.gitignore

```diff
@@ -109,3 +109,6 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# media files
```

Comment:
@usmandap please document via comment that this folder is only for dev mode

Reply:
done.